packages/loot-core/src/server/accounts/rules.ts

```diff
@@ -416,29 +416,34 @@ export class Condition {
   }
 }
 
-let ACTION_OPS = ['set', 'link-schedule'];
+export enum ActionOperator {
+  set = 'set',
+  linkschedule = 'link-schedule',
+}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** personally I prefer union types over enums. It would also allow you to do this migration with fewer changes.

https://www.becomebetterprogrammer.com/typescript-union-types-vs-enums/

But if you prefer to stick with enums - lets change it to camelCase then.
```suggestion
export enum ActionOperator {
  set = 'set',
  linkSchedule = 'link-schedule',
}
```

Reply:
Well using a type union is looking a whole lot simpler wowee. Let me know what you think of the revised changes, thanks 👍